CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ description: |-
 
 # VKCS Provider's changelog
 
+#### v0.1.12
+- Updated acceptance tests
```

Comment:
lets do not mention this since the change is not valuable for end-users

Reply:
Rewrote changelog